From 24784f56b9733ba48a997632d519ddd21f6c3840 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@freescale.com>
Date: Fri, 26 Apr 2013 13:02:38 +0530
Subject: [PATCH 0276/1383] fsl/usb: Add NULL entry in platform_device_id

Gadget compilation breaks without terminated NULL entry in platform_device_id

Signed-off-by: Suresh Gupta <suresh.gupta@freescale.com>
Change-Id: I5821f0bf73a3cdfdb7988079ac97237e9a3f5f8b
Reviewed-on: http://git.am.freescale.net:8181/2124
Reviewed-by: Li Yang-R58472 <LeoLi@freescale.com>
Reviewed-by: Mehresh Ramneek-B31383 <ramneek.mehresh@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/usb/gadget/udc/fsl_udc_core.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/udc/fsl_udc_core.c b/drivers/usb/gadget/udc/fsl_udc_core.c
index 2c417b6..a25b8e7 100644
--- a/drivers/usb/gadget/udc/fsl_udc_core.c
+++ b/drivers/usb/gadget/udc/fsl_udc_core.c
@@ -2674,7 +2674,9 @@ static const struct platform_device_id fsl_udc_devtype[] = {
 	}, {
 		.name = "imx-udc-mx51",
 	}, {
-		.name = "fsl-usb2-udc"
+		.name = "fsl-usb2-udc",
+	}, {
+		/* sentinel */
 	}
 };
 MODULE_DEVICE_TABLE(platform, fsl_udc_devtype);
-- 
2.8.1

